ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx

```diff
@@ -146,13 +148,20 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
         position={[9]}
       />
     ),
-    video: (props: ComponentToRenderProps) => (
-      <LegacyMediaPlayer
-        blocks={props.blocks}
-        className="mediaStyles"
-        css={styles.bodyMedia}
-      />
-    ),
+    video: (props: ComponentToRenderProps) =>
+      !isLive() ? (
+        <MediaLoader
+          blocks={props.blocks}
+          className="mediaStyles"
+          css={styles.bodyMedia}
+        />
+      ) : (
+        <LegacyMediaPlayer
+          blocks={props.blocks}
+          className="mediaStyles"
+          css={styles.bodyMedia}
+        />
+      ),
```

Comment:
Is it worth flipping this around and using `isLive` instead of checking the inverse? May just read a bit clearer for folks quickly parsing the file.